go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/Azure/azure-pipeline-go v0.2.2
 	github.com/Azure/azure-storage-blob-go v0.10.0
 	github.com/Azure/go-autorest/autorest/adal v0.9.1 // indirect
+	github.com/BurntSushi/toml v0.3.1
```

Comment:
Why the new deps?

Reply:
These were added when I was trying to i18n the go side. i did a `go mod tidy` to clean it up.
It should now be solved